src/Model/EventModel.php

```diff
@@ -40,10 +40,7 @@ public function __construct(array $config ) {
         );
         $this->properties = $config['properties'];
         // Can pass in unix timestamp if prefixed with '@'. Else just let it parse the date
-        $this->time = !empty($config['time']) ?
-            new DateTime(
-                is_int($config['time']) ? '@' . $config['time'] : $config['time']
-            ) : null;
+        $this->time = $config['time'];
```

Comment:
```suggestion
        $time = (array) new DateTime(
            is_int($config['time']) ? '@' . $config['time'] : $config['time']
        );
        
        $this->time = !empty($config['time']) ? $time['date'] : null;
```

Reply:
Won't this be a better approach? The logic basically takes care of epoch timestamps or any other incoming timestamps and then standardizes it

Reply:
This will fix the issue where it is currently returning a DateTime object for the time attribute